tests/psp1155/extensions/burnable.tests.ts

```diff
@@ -62,9 +62,9 @@ describe('MY_PSP1155_BURNABLE', () => {
         await expect(query.balanceOfBatch([[alice.address, token1], [alice.address, token2]]))
             .to.have.output([amount1, amount2])
```

Comment:
After updating of `burn` to allow burn without allowance you need to rewrite that test=)